README.md

```diff
@@ -2,14 +2,26 @@
 
 ### Usage
 
-This project is a proof of concept of a news feeder using NodeJS. It uses the Github API to fetch the relevant information from issues, releases..and then it generates a RSS feed with the latest news.
+As the Node.js organization continues to grow and expand, it becomes increasingly difficult to keep track of all the ongoing projects, teams, and working groups. We've recognized this challenge and have been actively seeking solutions to improve communication and awareness within our community.
```

Comment:
I think we want something here about helping people who report Node.js news to be able to find out about interesting news as that was the initial reason to work on the process.

Reply:
I added some changes in [cee0862](https://github.com/nodejs/nodejs-news-feeder/pull/26/commits/cee0862277d74663df5d23da06100e6906ba017d) that should cover that gap.